src/main/CommandLineSpecialNames.h

```diff
@@ -0,0 +1,12 @@
+#pragma once
```

Comment:
Maybe we can avoid adding a new file and use `Config.h` for this? We already have a few constants defined there like `MAXIMUM_QUORUM_NESTING_LEVEL`

Reply:
Makes sense! I switched to using `Config.h`.